### DISCUSSION TITLE
Can't find table(s) lemma_lookup for language 'la' in spacy-lookups-data. Make sure you have the package installed or provide your own lookup tables if no default lookups are available for your language.

### DISCUSSION THREAD

**Discussion by MartinaLeo** [explosion/spaCy#13559] [Help: Installation]
I'm trying to load the la_core_web_lg but I get the error above. Is it related to the recent commits and requires additional imports besides the one reported in https://spacy.io/universe/project/latincy?

**Comment by diyclassics** [comment]
Cf. Issues thread here: https://github.com/diyclassics/la_core_web_lg/issues/1
